crates/uv/tests/python_find.rs

```diff
@@ -455,4 +455,14 @@ fn python_find_unsupported_version() {
     ----- stderr -----
     error: No interpreter found for Python <3.0 in virtual environments or system path
     "###);
+
+    // Request free-threaded Python on unsupported version
+    uv_snapshot!(context.filters(), context.python_find().arg("2.12t"), @r###"
```

Comment:
Nit: We should probably use `3.12` here so it doesn't collide with our error message for unsupported Python versions if we ever move that.

Reply:
Oops you're right, this should be 3.12t.